src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -49,13 +89,151 @@ public Navigator() : base()
         {
             Layout = new AbsoluteLayout();
         }
+        
+        /// <summary>
+        /// An event for the page disappearing signal which can be used to subscribe or unsubscribe the event handler provided by the user.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<TransitionFinishedEventArgs> TransitionFinished;
 
         /// <summary>
         /// List of pages of Navigator.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public List<Page> NavigationPages { get; } = new List<Page>();
 
+        /// <summary>
+        /// Transition properties for the transition of View pair those have same transition tag.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Transition Transition
+        {
+            set
+            {
+                transition = value;
+            }
+            get
+            {
+                return transition;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ShowBehindPage
```

Comment:
How about not providing this property?

Because I am going to separate Page into ContentPage and DialogPage.
ContentPage is full screen page and DialogPage is dialog with dimmed background.

ContentPage does not show behind page when ContentPage is pushed.
DialogPage shows behind page when DialogPage is pushed.

Reply:
Currently, yes i will do it.